Makefile

```diff
@@ -85,3 +85,18 @@ build-tarball: clean man
 	find $(NAME)-$(VERSION) -name \*~ -exec rm {} \;
 	tar cfvj package/obs/$(NAME)-$(VERSION).tar.bz2 $(NAME)-$(VERSION)/
 	rm -rf $(NAME)-$(VERSION)/
+
+database-up:
+	docker-compose up db -d
```

Comment:
I can't comment on the rest of this PR, but I saw some conversation in our chat channels that `docker-compose` was deprecated in favor for `docker compose` =)

Reply:
aha old habits die hard, will update!

Reply:
done